services/regulated-assets-approval-server/internal/serve/tx_approve_test.go

```diff
@@ -166,27 +308,38 @@ func TestTxApproveHandler_isRejected(t *testing.T) {
 	require.NoError(t, err)
 	txEnc, err = tx.Base64()
 	req = txApproveRequest{
-		Transaction: txEnc,
+		Tx: txEnc,
 	}
-	rejectedResponse, err = handler.isRejected(ctx, req)
+	rejectedResponse, err = handler.txApprove(ctx, req)
 	require.NoError(t, err)
-	wantRejectedResponse = txApproveResponse{
-		Status: Sep8StatusRejected,
-		Error:  notImplementedErr,
+	wantRejectedResponse = txApprovalResponse{
+		Status:     "rejected",
+		Error:      "Not implemented.",
+		StatusCode: http.StatusBadRequest,
 	}
 	assert.Equal(t, &wantRejectedResponse, rejectedResponse)
 }
 
-func TestTxApproveHandler_serveHTTPJson(t *testing.T) {
+func TestAPIRejectedIntegration(t *testing.T) {
```

Comment:
nit: could we update our API tests naming so we use the underscore as proposed in [this post](https://ieftimov.com/post/testing-in-go-naming-conventions/#function-naming-conventions)? 

```suggestion
func TestAPI_RejectedIntegration(t *testing.T) {
```

Reply:
Addressed here 987bb8a